function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
```

Comment:
Consider naming the function a descriptive name to indicate what the function does